dask/dataframe/core.py

```diff
@@ -4310,7 +4310,7 @@ def set_index(
             See https://docs.dask.org/en/latest/dataframe-design.html#partitions
```

Comment:
```
        divisions: list, optional
            The "dividing lines" used to split the new index into partitions.
            For ``divisions=[0, 10, 50, 100]``, there would be three output partitions,
            where the new index contained [0, 10), [10, 50), and [50, 100), respectively.
            See https://docs.dask.org/en/latest/dataframe-design.html#partitions.
            If not given (default), good divisions are calculated by immediately computing
            the data and looking at the distribution of its values. For large datasets,
            this can be expensive.
            Note that if ``sorted=True``, specified divisions are assumed to match
            the existing partitions in the data; if this is untrue you should
            leave divisions empty and call ``repartition`` after ``set_index``.
```
Something like this might make the `divisions` parameter easier to understand for me, don't know if it would help anyone else?